policies.py

```diff
@@ -0,0 +1,84 @@
+import sonnet as snt
```

Comment:
Did you use `git mv` on this? I think if you do then the diff will show up properly.

Reply:
(Don't need to fix it now, just an fyi for the future.)

Reply:
I didn't use `git mv`, just `git add policies.py` and `git rm policy.py`. Not sure on best practices, so definitely open to feedback